aiida_common_workflows/workflows/relax/cp2k/generator.py

```diff
@@ -293,3 +322,23 @@ def _get_kpoints(kpoints_distance, structure, reference_workchain):
             kpoints_mesh.set_kpoints_mesh_from_density(distance=kpoints_distance)
             return kpoints_mesh
         return None
+
+    @staticmethod
+    def _get_cell_ref(structure, reference_workchain, scale_factor):
+        """If the reference_workchain specifies a CELL_REF, return that one, otherwise generate one"""
+
+        if reference_workchain and 'cp2k__parameters' in reference_workchain.inputs:
+            try:
+                return reference_workchain.inputs.cp2k__parameters['FORCE_EVAL']['SUBSYS']['CELL']['CELL_REF']
```

Comment:
```suggestion
                return reference_workchain.inputs.cp2k.parameters['FORCE_EVAL']['SUBSYS']['CELL']['CELL_REF']
```
I followed the kpoint example, but now I see a warning about the double underscore.